modules/sdk/gradle-plugins/CHANGELOG.markdown

```diff
@@ -129,6 +129,19 @@ version 1.0.125.
 - [LPS-66853]: Update the [Liferay Source Formatter] dependency to version
 1.0.275.
 
+## 2.0.29 - 2016-09-08
+
+### Changed
+- [LPS-66853]: Update the [Liferay Source Formatter] dependency to version
+1.0.276.
+
+## 2.0.30 - 2016-09-09
+
+### Added
+- [LPS-61099]: Allow to overwrite the default value of
+`liferay.appServerParentDir` by setting the project property
```

Comment:
Yes, it's the `appServerParentDir` property of the `liferay` [extension object](https://docs.gradle.org/current/userguide/custom_plugins.html#sec:getting_input_from_the_build).